### DISCUSSION TITLE
Request - Allow seed time tag without pausing the torrent

### DISCUSSION THREAD

**Discussion by SawkeeReemo** [StuffAnThings/qbit_manage#162] [Ideas]
As the subject says, I'd love to be able to automatically tag all torrents that have met their given seed-time requirements per tracker. Right now it seems we have that functionality, but it also pauses the torrent. For those of us who prefer to perma-seed, it would be great not to have to unpause things all the time.

Thanks!

**Comment by bakerboy448** [comment]
Achievable today, though not via a dedicated "tag only" toggle — it's a side effect of the upload-throttle option:
```yaml
share_limits:
  perma-seed:
    max_seeding_time: <your limit>
    cleanup: false
    upload_speed_on_limit_reached: -1
```
With `cleanup: false`, once a torrent hits its limit, qbm clears qBittorrent's share limits on that torrent (nothing left for qBittorrent to pause on) and applies `upload_speed_on_limit_reached` instead — `-1` means unlimited, so it keeps seeding uninterrupted. The group tag is applied independently of this at group-assignment time, so you get the tag without the pause.

Docs: Config-Setup.md, Share Limits section, `upload_speed_on_limit_reached` row + behavior note.

Separately: `share_limit_action: Default` (the default) defers to whatever qBittorrent's own global setting is — if that's set to pause/stop, that's what's pausing you today absent the throttle override above.
